Sources/UIBarButtonItem+Theme.m

```diff
@@ -26,28 +26,6 @@ + (UIBarButtonItem *)themedBackButtonWithTarget:(id)target andSelector:(SEL)sele
     return backButton;
 }
 
-+ (UIBarButtonItem *)themedRevealMenuButtonWithTarget:(id)target andSelector:(SEL)selector
-{
-    /* After day 354 of the year, the usual VLC cone is replaced by another cone
-     * wearing a Father Xmas hat.
-     * Note: this icon doesn't represent an endorsement of The Coca-Cola Company
-     * and should not be confused with the idea of religious statements or propagation there off
-     */
```

Comment:
I hope this amazing feature appears some place else now? 😆

Reply:
I hope so as well!

Reply:
But for later: "there off" is only one word: "thereof"